### PR TITLE
(IAC-1497) - Removal of Unsupported Translate Module

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: lib/puppet/provider/templates/webadministration/json_1.7.ps1.erb [hunk 1]
```diff
@@ -339,7 +339,7 @@ BEGIN {
   <xsl:template match="*[count(../*[name(../*)=name(.)])=count(../*) and count(../*)&gt;1]">
 -->
   <xsl:template match="*[count(../*[name(../*)=name(.)])=count(../*)]">
-    <xsl:variable name="inArray" select="translate(local-name(),'OBJECT','object')='object' or ../@Type[starts-with(.,'System.Collections') or contains(.,'[]') or (contains(.,'[') and contains(.,']'))]"/>
+    <xsl:variable name="inArray" select="local-name(),'OBJECT','object')='object' or ../@Type[starts-with(.,'System.Collections') or contains(.,'[]') or (contains(.,'[') and contains(.,']')]"/>
```
Comment (DavidS) [inline lib/puppet/provider/templates/webadministration/json_1.7.ps1.erb]: pretty sure this has nothing to do with the translate module, and will break the functionality of the module.

**Review by DavidS — CHANGES_REQUESTED** [review]

**Comment by DavidS** [comment]
Also please remove https://github.com/puppetlabs/puppetlabs-iis/tree/main/locales and check that no ruby code uses the `_()` underscore function

File: lib/puppet/provider/templates/webadministration/json_1.7.ps1.erb (hunk 1 above)
Comment (david22swan) [reply lib/puppet/provider/templates/webadministration/json_1.7.ps1.erb]: Yeh, wasn't really sure about this.
Will rip it out now.

**Comment by david22swan** [comment]
Have undone the template change and ripped out the locales folder.

**Review by DavidS — APPROVED** [review]

**Pull request merged**